pages/blueprintEdit/index.css

```diff
@@ -10,6 +10,11 @@
   --pf-c-pagination--m-bottom--BoxShadow: none;
 }
 
+.pf-c-options-menu.pf-m-top .pf-c-options-menu__menu {
+  --pf-c-options-menu__menu--Top: var(--pf-c-options-menu__menu--Top);
+  transform: translateY(0);
```

Comment:
@jgiardino, could you please fix this style lint issue? Thanks. :-)
```
pages/blueprintEdit/index.css

 15:3  ✖  Expected empty line before declaration  declaration-empty-line-before
```

Reply:
Thanks for catching that, @henrywang! I'm a little confused by this. This is the updated code that passes. Why is there a new line needed here?

```
.pf-c-options-menu.pf-m-top .pf-c-options-menu__menu {
  --pf-c-options-menu__menu--Top: var(--pf-c-options-menu__menu--Top);

  transform: translateY(0);
}
```